cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -404,44 +403,70 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             return parseInt(this.$('#due_in').val());
         },
 
+        showProjectedDate: function() {
+            var startDate = new Date(this.model.get('start'));
+            // The value returned by toUTCString() is a string in the form Www, dd Mmm yyyy hh:mm:ss GMT
+            var startDateList = startDate.toUTCString().split(' ')
+            this.$("#relative_weeks_due_start_date").text(startDateList[2] + ' ' + startDateList[1] + ', ' + startDateList[3]);
```

Comment:
Maybe add a comment above this line too to just give an example of what this will look like. Something like
```
Mmm dd, yyyy (i.e. Jul 26, 2021)
```
Also not the best that this will always put it in American formatting... but we can maybe try and tackle that separately